config/sync/field.field.config_pages.stanford_basic_site_settings.su_hide_ext_link_icons.yml

```diff
@@ -0,0 +1,23 @@
+uuid: cc184f07-8780-454f-80cd-91e72fea3300
+langcode: en
+status: true
+dependencies:
+  config:
+    - config_pages.type.stanford_basic_site_settings
+    - field.storage.config_pages.su_hide_ext_link_icons
+id: config_pages.stanford_basic_site_settings.su_hide_ext_link_icons
+field_name: su_hide_ext_link_icons
+entity_type: config_pages
+bundle: stanford_basic_site_settings
+label: 'Hide External Link Icons'
+description: 'Check this box to disable the external link icons.'
+required: false
+translatable: false
+default_value:
+  -
+    value: 0
```

Comment:
I think the default value should be "on". Can you adjust that in the UI and then re-export the configs.